docs/dev_guide/diagnostic_ref/dpct1047.rst

```diff
@@ -25,3 +25,47 @@ Suggestions to Fix
 
 If the matrix P is only used in library API, ignore this warning. If P is used
 other ways, you may need to adjust the value of P.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo(cublasHandle_t handle, float **a_array, float **b_array, int *p_array,
+             int *info_array) {
+      cublasSgetrfBatched(handle, 2, a_array, 2, p_array, info_array, 1);
+      cublasSgetrsBatched(handle, CUBLAS_OP_N, 2, 2, a_array, 2, p_array, b_array,
+                          2, info_array, 1);
+    }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo(dpct::queue_ptr handle, float **a_array, float **b_array, int *p_array,
+             int *info_array) {
+      /*
+      DPCT1047:0: The meaning of p_array in the dpct::getrf_batch_wrapper is
+      different from the cublasSgetrfBatched. You may need to check the migrated
+      code.
+      */
+      dpct::getrf_batch_wrapper(*handle, 2, a_array, 2, p_array, info_array, 1);
+      dpct::getrs_batch_wrapper(*handle, oneapi::mkl::transpose::nontrans, 2, 2,
+                                const_cast<float const **>(a_array), 2, p_array,
+                                b_array, 2, info_array, 1);
+    }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo(dpct::queue_ptr handle, float **a_array, float **b_array, int *p_array,
+             int *info_array) {
+      dpct::getrf_batch_wrapper(*handle, 2, a_array, 2, p_array, info_array, 1);

```

Comment:
Please add formula to explain why just removing the DPCT warning is OK.

Reply:
done